<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>flowTree</class>
 <widget class="QWidget" name="flowTree">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>899</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>系统名称：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="systemNameLab">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>测试项目：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="testProductLab">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="PfTreeWidget" name="treeWidget">
     <column>
      <property name="text">
       <string>流程名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>值</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>参数变化</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>启动条件</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>延时(ms)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>停止条件</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>发送时机</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>目标设备</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
  <action name="actionNewFlow">
   <property name="text">
    <string>新建流程</string>
   </property>
  </action>
  <action name="actionNewSubFlow">
   <property name="text">
    <string>新建子流程</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>复制</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPase">
   <property name="text">
    <string>粘贴</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>删除</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionTestSend">
   <property name="text">
    <string>测试发送</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PfTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>pftreewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
